Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -61,7 +61,7 @@ public SiteExtensionInfo(UIPackageMetadata data)
         public SiteExtensionInfo(UISearchMetadata data)
```

Comment:
any reason we can't call the `SiteExtensionInfo(UIPackageMetadata)` constructor with `data.LatestPackageMetadata` ? they seem to be doing the same thing

any reason this constructor can't just be 

```
```